### PR TITLE
Update README with setup instructions.

### PR DESCRIPTION
**Pull request by dellis23**
👓  @spladug 

Let me know if you want to move stuff around.  I didn't want to destroy your existing work, just augment it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,55 @@ If configured, graphite will be sent a metric of the form
...
+```
+https://github.com/reddit/rollingpin.git
+pip install .
+```
+
+Next, copy `example.ini` to `/etc/rollingpin.ini` or `~/.rollingpin.ini`, modifying the hostsource to suit your environment.
+
+For example, to set up an alias and deploy to 3 static hosts:
+
+```
```
Comment (spladug) [inline README.md]: if you pop "ini" on here github will syntax highlight nicely

File: README.md [hunk 2]
```diff
@@ -34,7 +34,55 @@ If configured, graphite will be sent a metric of the form
...
+[aliases]
+myhosts = host*
+```
+
+The hosts you are deploying to must have a local deploy script ready to be
+executed.  You can use `example-deploy.py` as a starting point.
+
+Once your deploy script is in place, update your configuration with its path:
+
+```
```
Comment (spladug) [inline README.md]: (same here)

File: README.md [hunk 3]
```diff
@@ -34,7 +34,55 @@ If configured, graphite will be sent a metric of the form
...
+For instance:
+
+```
+rollout -h myhosts -c foo
+```
+
+will call:
+
+```
+/usr/local/bin/deploy foo
```
Comment (spladug) [inline README.md]: it'll `sudo` too

File: README.md [hunk 4]
```diff
@@ -34,7 +34,55 @@ If configured, graphite will be sent a metric of the form
...
 
-See [example.ini](example.ini) for configuration instructions.
+Setup
+-----
+
+To install:
+
+```
+https://github.com/reddit/rollingpin.git
+pip install .
```
Comment (spladug) [inline README.md]: `python setup.py develop` ?

**Comment by spladug** [comment]
:nail_care: just minor stuff. looks great. thanks!

**Comment by dellis23** [comment]
💇

**Comment by dellis23** [comment]
💇

**Comment by spladug** [comment]
:tropical_fish:

**Pull request merged**